src/recipes/lavender-latte.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "Lavender Latte",
+    "description": "Lavender lattes are the perfect blend of calming, sweet, and creamy. They can be served hot or iced making them excellent for any type of weather!",
+    "github": "Shreya-L",
+    "ingredients": [
+        {
+            "quantity": "1/8-1/4",
+            "measure": "cup",
+            "ingredient": "dried lavender (can use lavender extract as well)"
+        },
+        {
+            "quantity": "1/2",
+            "measure": "cup",
+            "ingredient": "water"
+        },
+        {
+            "quantity": "1/4-1/2",
+            "measure": "cup",
+            "ingredient": "sugar"
+        },
+        {
+            "quantity": "1/2",
+            "measure": "cup",
+            "ingredient": "milk"
+        },
+        {
+            "quantity": "2/3",
+            "measure": "cup",
+            "ingredient": "strong brewed coffee"
+        },
+        {
+            "quantity": "1-2",
+            "measure": "tablespoons",
+            "ingredient": "lavender syrup"
+        },
+
+    ],
+    "directions": [
+        "1. Bring water to a boil, add dried lavender (or extract) and sugar",
+        "2. Stir sugar, lavender, water and reduce heat until liquid becomes more of a syrup",
+        "3. Add 1-2 tablespoons of lavender syrup to strong coffee and add milk to enjoy!"
```

Comment:
You can remove the numerations here. This array will be rendered as an ordered list.

Reply:
Ah I see that makes sense! Thank you!__